### PR TITLE
[opt](binlog) Support drop view binlog

### PR DESCRIPTION
**Pull request by smallx**
## Proposed changes

Support drop view binlog.

### CODE REVIEW

**Comment by smallx** [comment]
run buildall

**Comment by smallx** [comment]
@w41ter please help to review code, thank you.

File: fe/fe-core/src/main/java/org/apache/doris/binlog/DropTableRecord.java [hunk 1]
```diff
@@ -39,7 +41,10 @@ public DropTableRecord(long commitSeq, DropInfo info) {
         this.dbId = info.getDbId();
         this.tableId = info.getTableId();
         this.tableName = info.getTableName();
-        this.rawSql = String.format("DROP TABLE IF EXISTS `%s`", this.tableName);
+        this.isView = info.isView();
+        this.rawSql = info.isView()
+            ? String.format("DROP VIEW IF EXISTS `%s`", this.tableName)
+            : String.format("DROP TABLE IF EXISTS `%s`", this.tableName);
```
Comment (w41ter) [inline fe/fe-core/src/main/java/org/apache/doris/binlog/DropTableRecord.java:47]: It seems that apache doris hasn't DROP VIEW statement.

Comment (smallx) [reply fe/fe-core/src/main/java/org/apache/doris/binlog/DropTableRecord.java:47]: image

**Review by w41ter — DISMISSED** [review]

**Comment by w41ter** [comment]
run buildall

**Review by w41ter — APPROVED** [review]

**Comment by w41ter** [comment]
run buildall

**Comment by w41ter** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**